modules/sdk-core/src/bitgo/environments.ts

```diff
@@ -133,9 +133,9 @@ export const Environments: Environments = {
     uri: 'https://app.bitgo-latest.com',
     stellarFederationServerUrl: 'https://app.bitgo-latest.com/api/v2/xlm/federation',
   }),
-  staging: Object.assign({}, mainnetBase, {
-    uri: 'https://staging.bitgo.com',
-    stellarFederationServerUrl: 'https://staging.bitgo.com/api/v2/xlm/federation',
+  staging: Object.assign({}, devBase, {
+    uri: 'https://app.bitgo-staging.com',
+    stellarFederationServerUrl: 'https://app.bitgo-staging.com/api/v2/xlm/federation',
```

Comment:
Will need to reference `txlm` for testnet - `https://app.bitgo-staging.com/api/v2/txlm/federation`

Reply:
fixed now